### PR TITLE
fixes #5833 - suggested command do not work when created new trend counter

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by dmitri-d** [comment]
lgtm.

File: app/views/trends/index.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
    <p><strong><%= _("No trend counter found.") %></strong></p>
 
-   <p><%= ( _("To start collecting trend data, set a cron job to execute <span class='black'>RAILS_ENV=production bundle exec rake trends:counter</span> every Puppet Interval (%s minutes)") % Setting.puppet_interval).html_safe %>
+   <p><%= ( _("To start collecting trend data, set a cron job to execute <span class='black'>foreman-rake trends:counter</span> every Puppet Interval (%s minutes)") % Setting.puppet_interval).html_safe %>
```
Comment (GregSutcliffe) [inline app/views/trends/index.html.erb:21]: Line 11 needs this change too, it refers to only "rake" not "foreman-rake" and needs the html_safe adding to the Setting.

**Comment by lzap** [comment]
This was merged as https://github.com/theforeman/foreman/pull/1573 thanks!

**Comment by ohadlevy** [comment]
@lzap this does not look like it was merged?

**Comment by ohadlevy** [comment]
@isratrade ping

**Comment by isratrade** [comment]
fixed per @GregSutcliffe's comment

**Comment by dLobatog** [comment]
Merged as https://github.com/theforeman/foreman/commit/c08d8dbd8895319ac536547572536471da43df7f, @isratrade thanks for the update!

**Pull request closed without merging**